src/commands/PublishCommand.js

```diff
@@ -213,6 +213,10 @@ export default class PublishCommand extends Command {
     const patch = semver.inc(currentVersion, "patch");
     const minor = semver.inc(currentVersion, "minor");
     const major = semver.inc(currentVersion, "major");
+    const prepatch = semver.inc(currentVersion, "prepatch");
+    const preminor = semver.inc(currentVersion, "preminor");
+    const premajor = semver.inc(currentVersion, "premajor");
+    const prerelease = semver.inc(currentVersion, "prerelease");
```

Comment:
We should probably allow customization of the `identifier` argument to `semver.inc`, yes?

Otherwise the output is a little [terse](https://runkit.com/58b8a5649a21f300136b74aa/58b8a5649a21f300136b74ab):
```js
semver.inc("1.0.0", "prerelease")       // => 1.0.1-0
semver.inc("1.0.0", "prerelease", "rc") // => 1.0.1-rc.0
// etc
```